pkg/codegen/schema/schema.go

```diff
@@ -794,8 +795,8 @@ func (pkg *Package) marshalObjectData(
 	}, nil
 }
 
-func (pkg *Package) marshalObject(t *ObjectType) (ComplexTypeSpec, error) {
-	data, err := pkg.marshalObjectData(t.Comment, t.Properties, t.Language)
+func (pkg *Package) marshalObject(t *ObjectType, plain bool) (ComplexTypeSpec, error) {
```

Comment:
These changes are not really fixing a bug in prod code but just enabling the test right? This code does not run in prod contest? Could we move it in a file that only compiles for tests, or mark somehow? Always less spooky reviewing PRs that do not edit prod code.

Reply:
This code could run in prod, but nothing currently uses it. It is generally useful code, so I'd prefer to leave it where it is and add tests for it specifically.